### DISCUSSION TITLE
NestJS Platform Adapter

### DISCUSSION THREAD

**Discussion by smolinari** [kartikk221/hyper-express#218] [Ideas]
Hi everyone,

I'd just like to make the community here aware of our budding efforts to make a [Hyper Express adapter for NestJS](https://github.com/m8a-io/hyper-express-adapter) to add it as a platform along with Express and Fastify. The intention is to use Hyper Express as a Websocket platform too. I'm super excited to make this happen and if we can get it going, the relation to NestJS would probably be a big boost to this community too. 

What do you think? Do let me know. 

Oh, and any support/ contributions to the effort are highly welcome!!! 

Scott

**Comment by kartikk221** [comment]
Hey, this is a great idea. I would love to help out wherever possible in underlying APIs / Functionalities.

**Reply by smolinari** [reply]
Awesome!!! I was hoping we could collaborate. :) It's still very new/ in its infancy. But, I'm very hopeful!

Scott

**Reply by smolinari** [reply]
We have "hello world" working. :grin: 

Scott

**Reply by kartikk221** [reply]
> Awesome!!! I was hoping we could collaborate. :) It's still very new/ in its infancy. But, I'm very hopeful!
> 
> 
> 
> Scott

I haven't personally worked with NestJS in the past but I could help out with any features in isolation as needed.

**Reply by smolinari** [reply]
FYI, Hyper-Express is killing it in this performance comparison vs. .Net. 

https://medium.com/deno-the-complete-reference/net-vs-node-js-frameworks-hello-uuid-performance-c970a5d8548a

I also added a comment to try and drum up help to get Hyper-Express to feature parity with Express. I hope you are ok with that. 

Scott

**Reply by smolinari** [reply]
I believe it would be nice to also list out in issues which features aren't in parity or missing. That would give external devs something to chew and work on. :grin: 

Scott
